packages/nep141-erc20/package.json

```diff
@@ -25,8 +25,10 @@
     "eth-object": "near/eth-object#54e03b8aac8208cf724e206d49ffb8bdd30451d7",
     "eth-revert-reason": "^1.0.3",
     "eth-util-lite": "near/eth-util-lite#master",
```

Comment:
Which dependency can we get rid of?

Reply:
Nothing for now. merle-patricia-tree was not defined explicitly in the package.json and we still want access to the old tree just in case some comparison is needed. This doesn't prevent solving the browser support issue as long as the dependency is not imported.